src/main/webapp/app/localci/build-queue/build-queue.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { DataTableComponent } from 'app/shared/data-table/data-table.component';
+import { AccountService } from 'app/core/auth/account.service';
+import { BuildQueueService } from 'app/localci/build-queue/build-queue.service';
+import { BuildJob } from 'app/entities/build-job.model';
+import { faRefresh } from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+    selector: 'jhi-build-queue',
+    templateUrl: './build-queue.component.html',
+    styleUrl: './build-queue.component.scss',
+})
+export class BuildQueueComponent implements OnInit {
+    @ViewChild(DataTableComponent) dataTable: DataTableComponent;
+
+    rowClass: string | undefined = undefined;
```

Comment:
I think `rowClass?: string | undefined;` is cleaner, correct me if I'm wrong

Reply:
`rowClass?: string;` should be sufficient